### PR TITLE
use the config for the next requests (+ cookies)

### PR DESCRIPTION
**Pull request by jarnix**
For the issue #108 .

I just added a new service for holding the config and then added a request handler on the http interceptor to read the config and send it with every request.

An update from #109 that I closed, because angular was not sending the cookies anymore...

### CODE REVIEW

**Comment by witoldsz** [comment]
I am not sure if creating separate configuration service is the right approach. Maybe the module itself should be configurable by it's own provider, like for example the $http:
https://docs.angularjs.org/api/ng/provider/$httpProvider

File: src/http-auth-interceptor.js [hunk 1]
```diff
@@ -10,7 +10,18 @@
 
   angular.module('http-auth-interceptor', ['http-auth-interceptor-buffer'])
 
-  .factory('authService', ['$rootScope','httpBuffer', function($rootScope, httpBuffer) {
+  	/**
```
Comment (witoldsz) [inline src/http-auth-interceptor.js]: This formatting is a mess. You have to adapt to the formatting of this file or it'll never be merged.

File: src/http-auth-interceptor.js [hunk 2]
```diff
@@ -10,7 +10,18 @@
...
+	 */
+	.factory('authConfigService', function() {
+		return {
+			config : {
+				headers: {}
+			}
+		};
+	})
+  
+  .factory('authService', ['$rootScope','httpBuffer', 'authConfigService', function($rootScope, httpBuffer, authConfigService) {
```
Comment (witoldsz) [inline src/http-auth-interceptor.js]: Some lines (like this one) are so long I can't even see what's there in GitHub's default code review page.

**Comment by jarnix** [comment]
nevermind

**Pull request closed without merging**